README.md

```diff
@@ -1,10 +1,10 @@
 # ChargeIO for WooCommerce
 
-Plugin version: 1.2
+Plugin version: 1.3
 
 Requires at least:
 - WordPress 3.8.0
```

Comment:
We should probably update the minimum WordPress version to 4.4 to match the WooCommerce requirements.